tests/performance/test_performance.py

```diff
@@ -54,15 +63,177 @@ def test_performance(config_path):
     with open(config_path, 'r') as config_file:
         config = json.load(config_file)
 
-    transformer = get_instance(config['transformer'], **config['kwargs'])
-    dataset_gen = get_instance(config['dataset'])
+    transformer_instance = get_instance(config['transformer'], **config['kwargs'])
+    dataset_generator = get_instance(config['dataset'])
 
-    out = profile_transformer(transformer, dataset_gen, config['transform_size'],
-                              config['fit_size'])
+    out = profile_transformer(
+        transformer=transformer_instance,
+        dataset_generator=dataset_generator,
+        fit_size=config['fit_size'],
+        transform_size=config['transform_size'],
+    )
 
     assert out['Fit Time'] < config['expected']['fit']['time']
     assert out['Fit Memory'] < config['expected']['fit']['memory']
     assert out['Transform Time'] < config['expected']['transform']['time']
     assert out['Transform Memory'] < config['expected']['transform']['memory']
     assert out['Reverse Transform Time'] < config['expected']['reverse_transform']['time']
     assert out['Reverse Transform Memory'] < config['expected']['reverse_transform']['memory']
+
+
+def _round_to_magnitude(value):
+    for digits in range(-15, 15):
+        rounded = np.round(value, digits)
+        if rounded != 0:
```

Comment:
I wouldn't expect this to happen so this should be safe, but is it at all possible for the value to be to small or actually 0?

Reply:
I would not expect it to happen either, but if it happens I would assume that getting a `ValueError` would be the proper behavior, since if the `value` is 0 we cannot really set any reasonable upper bound following the current logic.
I'll update the function to raise the proper exception message if that's the case, and if it ever becomes a blocker in the future we can always revisit it.